docs/faq/network/node-security.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_label: 'Security'
+sidebar_position: 6
+---
+
+# Node Security
+
+### If I run a node, can my location be tracked?
+
+Yes, your location can be tracked by running a node, as your node machine's public IP address will be exposed to connect to other peers within the blockchain network.
+
+### Is it possible to get access to my funds from the machine?
+
+If someone could log into your node, he can **only access your validator keys if he also knows your validator wallet password**. By gaining access to your wallet password, the attacker could act maliciously by forcing slashable events or restarting the client with a different recipient fee address.
+
+Machine access is risky if you set up automation with a password file or use Lighthouse- as the client stores the wallet password within the config file during the initial startup. Therefore, it's a must-have to restrict access to your node **using key authentication** and **restrict read and write access of files to a specific user**. This way, no one can see or change your configurations without also knowing the admin password of your machine. You can find more information about all the security setups within the [Extended Node Guide](https://docs.luksoverse.io/docs/) written by community members.
```

Comment:
"it's a must-have to restrict access to your node **using key authentication** and **restrict read and write access of files to a specific user**" -> replace with

it's a must-have to restrict access to your node **using key authentication** and **read and write access to files for a specific user**